src/iop/lut3d.c

```diff
@@ -790,6 +790,13 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
   }
 }
 
+void filepath_set_OS_separator(char *filepath)
```

Comment:
Maybe change the name now :) : filepath_set_unix_separator

Reply:
It is not only for unix ... the goal is to be ok on the 3 main OS.
Would be filepath_set_all_OS_separator() convenient ?

Reply:
No, not ok. The point is that the routine must be named to convey the actual implementation. The implementation is to change dir separators to UNIX style. So filepath_set_unix_separator() looks perfect to me.

Reply:
Question of standpoint. Not a big deal. I do it.